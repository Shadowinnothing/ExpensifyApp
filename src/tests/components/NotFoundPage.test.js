import React from 'react';
import {shallow} from 'enzyme';

import uNotFoundPage from '../../components/NotFoundPage';

describe('components/NotFoundPage', () => {

  test('should correctly render NotFoundPage', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });

});
