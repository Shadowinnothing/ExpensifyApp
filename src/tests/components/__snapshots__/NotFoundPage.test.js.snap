// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NotFoundPage should correctly render NotFoundPage 1`] = `
<div>
  <h1>
    404 Not Found
  </h1>
  <YouTube
    className={null}
    containerClassName=""
    id={null}
    onEnd={[Function]}
    onError={[Function]}
    onPause={[Function]}
    onPlay={[Function]}
    onPlaybackQualityChange={[Function]}
    onPlaybackRateChange={[Function]}
    onReady={[Function]}
    onStateChange={[Function]}
    opts={
      Object {
        "height": "390",
        "playerVars": Object {
          "autoplay": 1,
        },
        "width": "640",
      }
    }
    videoId="ZZ5LpwO-An4"
  />
  <br />
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
